### ISSUE TITLE
Render crash

### ISSUE THREAD

**Issue by asiekierka** [Victorious3/Integrated-Circuits#68]
````
java.lang.NullPointerException
	at vic.mod.integratedcircuits.client.TileEntityGateRenderer.renderWorldBlock(TileEntityGateRenderer.java:26) ~[TileEntityGateRenderer.class:?]
	at cpw.mods.fml.client.registry.RenderingRegistry.renderWorldBlock(RenderingRegistry.java:118) ~[RenderingRegistry.class:?]
	at FMLRenderAccessLibrary.renderWorldBlock(FMLRenderAccessLibrary.java:53) ~[FMLRenderAccessLibrary.class:?]
	at net.minecraft.client.renderer.RenderBlocks.func_147805_b(RenderBlocks.java:296) ~[blm.class:?]
	at net.minecraft.client.renderer.WorldRenderer.func_147892_a(WorldRenderer.java:181) ~[blo.class:?]
	at fastcraft.a.o.a(F:101) ~[fastcraft-1.21.jar:?]
	at fastcraft.Hooks.t(F:147) ~[fastcraft-1.21.jar:?]
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1187) ~[blt.class:?]
	at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1011) ~[blt.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:989) ~[bao.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887) [bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_31]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_31]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_31]
	at java.lang.reflect.Method.invoke(Method.java:483) ~[?:1.8.0_31]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.11.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.11.jar:?]
````

**Comment by Victorious3** [comment]
I don't know why it thinks it is a good idea to render TEs before they were fully initialized... At this point I should create a new release with the bug fixes for 0.8.

**Comment by Victorious3** [comment]
Going to reopen because I need the same fix in 0.9.
